src/window.rs

```diff
@@ -961,6 +966,15 @@ impl Default for CursorIcon {
     }
 }
 
+#[derive(Clone)]
+pub struct CursorRgba {
+    pub xhot: u16,
```

Comment:
Extent and offset related parts of the API should use `dpi::{PhysicalPosition, PhysicalSize}` or `dpi::{LogicalPosition, LogicalSize}` to make it dpi aware. (I haven't checked so far how scaling logic applies in this case)

Reply:
Done